buildbot-host/rebuild-all.sh

```diff
@@ -17,4 +17,4 @@ for DOCKERFILE in $(find -maxdepth 2 -type f -regextype egrep -regex '.*/(Docker
 done
 
 # clean up dangling images and build cache
-docker system prune --force
+docker image prune --force
```

Comment:
We did not change this for fun. image does not clean up enough. It sounds like it should, but a lot of stuff remains in overlays, presumably the build cache that docker system prunes.

I never encountered the network problem since we always have a buildmaster running.

Reply:
Another option is to recreate the buildbot-net network at the end of rebuild-all.sh - that's what I did to work around this problem. Not pretty, but it does work.

Reply:
I think the solution here is to replace docker system prune with the individual commands. So

```suggestion
docker container prune --force
docker image prune --force
docker buildx prune --force
```

This should be equivalent to docker system prune without touching networks.

Reply:
I tested the proposed change and it seemed to work. Applying the changes.